### DISCUSSION TITLE
curiosity

### DISCUSSION THREAD

**Discussion by apakose-ezekiel** [Cyfrin/foundry-full-course-cu#1848] [Q&A]
Am new to the coding world and i hope this is the right course for me?

**Comment by urosognjenovic** [comment]
This is probably the best Foundry course you'll find. It starts from scratch so you'll have no issues understanding all the concepts.

**Reply by 0xodus-eth** [reply]
Agreed

**Comment by Ifra001** [comment]
From my experience, I have taken I few coding lessons before this one so the coding part of the course at this point seems pretty manageable and easy so far for me. Patrick is a great explainer and he helps me understand difficult concepts easily! (He is also motivational and funny at the same time so you will have fun taking this course! :) )

**Reply by EngrPips** [reply]
Indeed, Patrick did a great job explaining stuff.
